### ISSUE TITLE
No empty lines between closing braces

### ISSUE THREAD

**Issue by tp** [eslint/eslint#5195]
I would like to propose a rule that would flag the following cases:

```js
let x = {
  y: {
    z: 1,
  },

};
```

```js
class X {
  f() {

  }

}
````

In both cases the second to last line should be an error in my opinion.

I was unable to configure this using the existing rules.

**Comment by eslintbot** [comment]
@tp Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by ilyavolodin** [comment]
Could you feel out our [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) template please? It makes it easier for everyone to see all of the information in the same format. Thanks!

**Comment by alberto** [comment]
There is a proposal for classes in #5092

**Comment by nzakas** [comment]
No response, so closing.
